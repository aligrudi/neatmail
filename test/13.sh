# neatmail test
./mail pg -m $1 =msg003
