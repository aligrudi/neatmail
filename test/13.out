From root Tue Dec 23 03:03:03 2015
From: C <c@c.eu>
To: A <a@a.eu>
Cc: B <b@b.eu>
Subject: Message from C
Message-Id: <msg003>
Date: Tue, 23 Dec 2015 03:03:00 +0100

This is C.

